examples/credit-risk-end-to-end/01_Credit_Risk_Data_Prep.ipynb

```diff
@@ -0,0 +1,1103 @@
+    "train_y.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "06406ee2-df4f-4dbb-bff0-a592a8e3da37",
+   "metadata": {},
+   "source": [
+    "To verify that we have introduced a lag for the class outcome, check the `outcome timestamp` for the 5 records above against the `application_timestamp` further up. You should see a difference of 30 to 90 days."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e720ce24-e092-4fcd-be3e-68bb18f4d2a7",
+   "metadata": {},
+   "source": [
+    "### Save Datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5cae0578-8431-46c7-8d64-e52146f47d46",
+   "metadata": {},
+   "source": [
+    "Now that we have our features and target data prepared, let's save it to local parquet files in the `data` directory (parquet is one of the file formats supported by Feast). \n",
+    "\n",
+    "One more step we will add is splitting the context data columnwise and saving it in two files. This step is contrived--we don't usually split data when we don't need to--but it will allow us to demonstrate later how Feast can easily join datasets (a common need in Data Science projects)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "cebef56c-1f54-4d31-a545-75d708d38579",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the data directory if it doesn't exist\n",
+    "os.makedirs(\"Feature_Store/data\", exist_ok=True)\n",
+    "\n",
+    "# Split and save data\n",
+    "a_cols = ['ID', 'duration', 'credit_amount', 'installment_commitment', 'application_timestamp']\n",
+    "b_cols = ['ID', 'residence_since', 'age', 'existing_credits', 'num_dependents','application_timestamp']\n",
+    "\n",
+    "train_num[a_cols].to_parquet(\"Feature_Store/data/train_a.parquet\", engine=\"pyarrow\")\n",
+    "train_num[b_cols].to_parquet(\"Feature_Store/data/train_b.parquet\", engine=\"pyarrow\")\n",
+    "test_num.to_parquet(\"Feature_Store/data/test.parquet\", engine=\"pyarrow\")\n",
+    "\n",
+    "train_y.to_parquet(\"Feature_Store/data/train_y.parquet\", engine=\"pyarrow\")\n",
+    "test_y.to_parquet(\"Feature_Store/data/test_y.parquet\", engine=\"pyarrow\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8d5de9f-bd27-4e95-802c-b121743dd1b0",
+   "metadata": {},
+   "source": [
+    "We have saved the following files to the `Feature_Store/data` directory: \n",
+    "- `train_a.parquet` (training data, a columns)\n",
+    "- `train_b.parquet` (training data, b columns)\n",
+    "- `train_y.parquet` (training outcomes)\n",
+    "- `test.parquet` (test data)\n",
+    "- `test_y.parquet` (test outcomes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af6355dc-ff5b-4b3f-b0bd-3c4020ef67e8",
+   "metadata": {},
+   "source": [
+    "With the feature data prepared, we are ready to setup and deploy the feature store. \n",
+    "\n",
+    "Continue with the [02_Credit_Risk_Model_Training.ipynb](02_Credit_Risk_Model_Training.ipynb) notebook."
```

Comment:
This link is not working for me when i run the notebook on local and also on RHOAI platform.

Reply:
I confirm, it should be [02_Deploying_the_Feature_Store](02_Deploying_the_Feature_Store.ipynb)

Reply:
Fixed.